src/api.js

```diff
@@ -55,14 +56,18 @@ const api = {
     const { siteId, seriesSlug, chapterSlug } = utils.getIdComponents(id);
     return instance.get(`/chapter`, {
       params: { siteId, seriesSlug, chapterSlug },
+      timeout: SCRAPING_TIMEOUT,
```

Comment:
If we try scraping for longer than 8 seconds, we abandon the request. This change isolates the 8s timeout to scraping endpoints only. I experienced a few timeouts from loading a collection, which shouldn't happen.